clippy_lints/src/collapsible_match.rs

```diff
@@ -170,3 +174,19 @@ fn is_none_ctor(res: Res, tcx: TyCtxt<'_>) -> bool {
     }
     false
 }
+
+/// Gets a binding ID with optional `&` or `*` operators (e.g. `&**foo`)
```

Comment:
"with optional `&` or `*`  removed?"

Reply:
Yes. I can improve the comments here. I don't think this should be in utils because of the nuance with deref. For example, a deref of `&Vec -> Vec` is allowed but a deref of `Vec -> [_]` will yield `None`.

Reply:
Sounds good. r=me with comments

Reply:
So what exactly does r=me mean? Is that speaking to bors?

Reply:
bors and I have not been formally introduced 😆

Reply:
It basically means, make the requested change and then write `@bors r=Manishearth` (or the name of the reviewer). Usually the reviewer forgets to give r+ rights to the author though 😄

Reply:
and then what does that do?

Reply:
I found the [docs](https://forge.rust-lang.org/compiler/reviews.html)!